other/install_2-1_mysql.sql

```diff
@@ -1718,7 +1718,7 @@ VALUES ('xx', 'Standard', '0'),
 
 INSERT INTO {$db_prefix}messages
 	(id_msg, id_msg_modified, id_topic, id_board, poster_time, subject, poster_name, poster_email, poster_ip, modified_name, body, icon)
-VALUES (1, 1, 1, 1, UNIX_TIMESTAMP(), '{$default_topic_subject}', 'Simple Machines', 'info@simplemachines.org', '127.0.0.1', '', '{$default_topic_message}', 'xx');
+VALUES (1, 1, 1, 1, UNIX_TIMESTAMP(), '{$default_topic_subject}', 'Simple Machines', 'info@simplemachines.org', '', '', '{$default_topic_message}', 'xx');
```

Comment:
an emtpy string is no valid ip address,
please fill null in this field,
for this you got two ways:
first:
take the poster_ip field out from the insert statement plus the ''
or:
replace '' with null

recommended way is the first one because is use the default value of the field in the database.